index.js

```diff
@@ -4,6 +4,17 @@ var transpiler = require('babel-core');
 var Filter     = require('broccoli-filter');
 var clone      = require('clone');
 
+function replaceExtensions(extensions, name) {
+  var regex;
+
+  for (var i = 0, l = extensions.length; i < l; i++) {
+    regex = new RegExp('\.' + extensions[i], 'g');
```

Comment:
these regexp:
* shouldn't be greedy
* should end with `$/`
* should be computed once in the constructor.